api_app/models/schemas/airlock_request.py

```diff
@@ -27,15 +27,33 @@ class Config:
         }
 
 
+class AirlockRequestWithAllowedUserActions(BaseModel):
```

Comment:
This seems odd to me - why do we need it as a field? It's supposed to be in the user role...